planner/go_planner.go

```diff
@@ -39,6 +51,31 @@ func (g *GoPlanner) GetPlan(srcDir string) *Plan {
 	}
 }
 
+func getGoPackage(srcDir string) string {
+	goModPath := filepath.Join(srcDir, "go.mod")
+	goVersion := getVersion(goModPath)
+	v, ok := versionMap[goVersion]
+	if ok {
+		return v
+	} else {
+		// Should we be throwing an error instead, if we don't have a nix package
+		// for the specified version of go?
```

Comment:
Might be a good idea if we don't have a newer enough version.

Reply:
Decided to keep: as long as we're updating this to always have the latest version of go, it should work ok. Later we can parse the versions and compare, so we only give the error if the version is greater.